src/harness/certs/revoke_and_generate_crl.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash 
+
+mkdir -p crl
+
+function blacklist_certificate()
+{
+  #Argument1: Name of the certificate to be  blacklisted.
+  #Argument2: Intermediate CA to generate CRL.
+
+  # Revoke certificate.
+  openssl ca -revoke $1 -keyfile private/$2.key -cert $2.cert \
+      -config ../../../cert/openssl.cnf
+  generate_crl_chain
+}
+
+function generate_crl_chain()
+{
+  #Create a CRL for root CA containing the revoked intermediate CA certificate.
+  echo "\n\n Generate CRL for root_ca"
```

Comment:
These need `echo -e` for the formatting to work:

    $ echo "\ntest"
    \ntest
    $ echo -e "\ntest"
    
    test

(Also update the corresponding `\n`s in CRLServer-README.md)

Reply:
thanks, fixed all places.